galaxykit/client.py

```diff
@@ -132,6 +133,7 @@ def __init__(
                     container_registry,
                     tls_verify=container_tls_verify,
                 )
+        self.rbac_enabled = self._is_rbac_available()
```

Comment:
yeah so because galaxykit.greet creates a GalaxyClient object but there's no real `galaxy_ng` instance that it's connecting to, this function call will cause CI to fail.

So the solution here is to, like @ironfroggy suggested, place the call in `_http()`, in a conditional that only calls it if `self.rbac_enabled` is empty.